nucleus/scene.py

```diff
@@ -15,10 +15,11 @@
 from .dataset_item import DatasetItemType, DatasetItem
 
 
-@dataclass
 class Frame:
```

Comment:
Don't skip making this a dataclass, just add keyword args besides items to the items dict

Reply:
The downside to this approach is you lose __repr__, eq and many other convenience functions